### PR TITLE
[bmvi.de] New ruleset

### PR DESCRIPTION
**Pull request by rugk**
BTW for all German readers: [They were affected by Heartbleed until Thursday](http://www.heise.de/security/meldung/Ministerium-fuer-digitale-Infrastruktur-pfuscht-beim-eigenen-Web-Server-3186960.html).

### CODE REVIEW

File: src/chrome/content/rules/bmvi.de.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!--
+    Nonfunctional subdomain:
+        - $
+-->
+<ruleset name="Bundesministerium für Verkehr und digitale Infrastruktur">
+    <target host="www.bmvi.de/" />
```
Comment (fuglede) [inline src/chrome/content/rules/bmvi.de.xml]: This one should be <code>www.bmvi.de</code> (without the slash).

Comment (rugk) [inline src/chrome/content/rules/bmvi.de.xml]: Fixed

**Comment by J0WI** [comment]
Please also document broken hosts for future maintenance:
bis.bmvi.de
jobboerse.bmvi.de
m.bmvi.de

**Comment by rugk** [comment]
Done

**Pull request merged**